backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5687500_sys_M_HU_Trace_Backup.sql

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Is this needed? I was expecting the script to also manipulate the `M_HU_Trace` table.